packages/components/src/FormField/FormField.tsx

```diff
@@ -146,145 +138,150 @@ export interface FormFieldProps {
   onBlur?(): void;
 
   /**
-   * **DEPRECATED** Use `onValidation` prop instead.
-   *
    * Callback to get the the status and message when validating a field
-   * @param status
-   * @param message
+   * @param messages
    */
-  onValidate?(status: "pass" | "fail", message: string): void;
+  onValidation?(message: string | undefined): void;
```

Comment:
Is `message` undefined if the input is valid?

Reply:
I think this was in here when I was learning. Not needed